From c93d58283121c6aed44d137d88f5837f6995bbee Mon Sep 17 00:00:00 2001
From: Dominik 'Rathann' Mierzejewski <dominik@greysector.net>
Date: Mon, 6 Oct 2025 22:05:53 +0200
Subject: [PATCH 1/2] add missing header include for QElapsedTimer class

This is required in order to use the `QElapsedTimer` class:
https://doc.qt.io/qt-6/qelapsedtimer.html

Fixes #724 .
---
 src/plugins/vumeter/vumeterwidget.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/vumeter/vumeterwidget.cpp b/src/plugins/vumeter/vumeterwidget.cpp
index 786f24b5b..7ccd3db06 100644
--- a/src/plugins/vumeter/vumeterwidget.cpp
+++ b/src/plugins/vumeter/vumeterwidget.cpp
@@ -32,6 +32,7 @@
 
 #include <QActionGroup>
 #include <QBasicTimer>
+#include <QElapsedTimer>
 #include <QContextMenuEvent>
 #include <QJsonObject>
 #include <QMenu>

From d4ee42db2c8514a488218d97aca1348a780c679b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luke=20Taylor=20=7C=20=E5=8D=A2=E7=86=99=E8=AF=9A?=
 <luket@pm.me>
Date: Mon, 6 Oct 2025 23:25:50 +0100
Subject: [PATCH 2/2] Fix formatting

---
 src/plugins/vumeter/vumeterwidget.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/vumeter/vumeterwidget.cpp b/src/plugins/vumeter/vumeterwidget.cpp
index 7ccd3db06..2834a3e19 100644
--- a/src/plugins/vumeter/vumeterwidget.cpp
+++ b/src/plugins/vumeter/vumeterwidget.cpp
@@ -32,8 +32,8 @@
 
 #include <QActionGroup>
 #include <QBasicTimer>
-#include <QElapsedTimer>
 #include <QContextMenuEvent>
+#include <QElapsedTimer>
 #include <QJsonObject>
 #include <QMenu>
 #include <QPainter>
